device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/td2-a7050-qx32-32x40G.config.bcm

```diff
@@ -870,3 +478,531 @@ serdes_preemphasis_lane2_32=0x8fc0
+xgxs_tx_lane_map_93.0=0x3120
+
+# Port {xe96} [40G only]
+phy_an_c37_97.0=3
+phy_an_c73_97.0=1
+phy_xaui_rx_polarity_flip_97.0=0x0
+phy_xaui_tx_polarity_flip_97.0=0x0
+port_init_autoneg_97.0=0
+port_phy_addr_97.0=0x7f
+portmap_97.0=61:40
+serdes_firmware_mode_97.0=2
+xgxs_rx_lane_map_97.0=0x2031
+xgxs_tx_lane_map_97.0=0x2031
+
+# Port {xe97} [40G only]
+phy_an_c37_98.0=3
+phy_an_c73_98.0=1
+phy_xaui_rx_polarity_flip_98.0=0x0
+phy_xaui_tx_polarity_flip_98.0=0x0
+port_init_autoneg_98.0=0
+port_phy_addr_98.0=0x7f
+portmap_98.0=57:40
+serdes_firmware_mode_98.0=2
+xgxs_rx_lane_map_98.0=0x2031
+xgxs_tx_lane_map_98.0=0x2031
+
+# Port {xe98} [40G only]
+phy_an_c37_99.0=3
+phy_an_c73_99.0=1
+phy_xaui_rx_polarity_flip_99.0=0x0
+phy_xaui_tx_polarity_flip_99.0=0x0
+port_init_autoneg_99.0=0
+port_phy_addr_99.0=0x7f
+portmap_99.0=81:40
+serdes_firmware_mode_99.0=2
+xgxs_rx_lane_map_99.0=0x0213
+xgxs_tx_lane_map_99.0=0x0213
+
+# Port {xe99} [40G only]
+phy_an_c37_100.0=3
+phy_an_c73_100.0=1
+phy_xaui_rx_polarity_flip_100.0=0x0
+phy_xaui_tx_polarity_flip_100.0=0x0
+port_init_autoneg_100.0=0
+port_phy_addr_100.0=0x7f
+portmap_100.0=85:40
+serdes_firmware_mode_100.0=2
+xgxs_rx_lane_map_100.0=0x0213
+xgxs_tx_lane_map_100.0=0x0213
+
+# Port {xe100} [40G only]
+phy_84328_101.0=1
+phy_an_c37_101.0=3
+phy_an_c73_101.0=1
+phy_aux_voltage_enable_101.0=0x1
+phy_ext_rom_boot_101.0=0x0
+phy_line_tx_mode_101.0=1
+phy_rx_polarity_flip_101.0=0x0
+phy_system_tx_mode_101.0=0
+phy_tx_polarity_flip_101.0=0x0
+phy_xaui_rx_polarity_flip_101.0=0x0
+phy_xaui_tx_polarity_flip_101.0=0x0
+port_phy_addr_101.0=0x54
+port_phy_clause_101.0=0x2d
+port_phy_id0_101.0=0x600d
+port_phy_id1_101.0=0x8500
+portmap_101.0=117:40
+serdes_firmware_mode_101.0=1
+xgxs_rx_lane_map_101.0=0x0123
+xgxs_tx_lane_map_101.0=0x0123
+
+# Port {xe101} [40G only]
+phy_84328_102.0=1
+phy_an_c37_102.0=3
+phy_an_c73_102.0=1
+phy_aux_voltage_enable_102.0=0x1
+phy_ext_rom_boot_102.0=0x0
+phy_line_tx_mode_102.0=1
+phy_rx_polarity_flip_102.0=0x0
+phy_system_tx_mode_102.0=0
+phy_tx_polarity_flip_102.0=0x0
+phy_xaui_rx_polarity_flip_102.0=0x0
+phy_xaui_tx_polarity_flip_102.0=0x0
+port_phy_addr_102.0=0x50
+port_phy_clause_102.0=0x2d
+port_phy_id0_102.0=0x600d
+port_phy_id1_102.0=0x8500
+portmap_102.0=113:40
+serdes_firmware_mode_102.0=1
+xgxs_rx_lane_map_102.0=0x0123
+xgxs_tx_lane_map_102.0=0x0123
+
+# Port {xe102} [40G only]
+phy_84328_103.0=1
+phy_an_c37_103.0=3
+phy_an_c73_103.0=1
+phy_aux_voltage_enable_103.0=0x1
+phy_ext_rom_boot_103.0=0x0
+phy_line_tx_mode_103.0=1
+phy_rx_polarity_flip_103.0=0x0
+phy_system_tx_mode_103.0=0
+phy_tx_polarity_flip_103.0=0x0
+phy_xaui_rx_polarity_flip_103.0=0x0
+phy_xaui_tx_polarity_flip_103.0=0x0
+port_phy_addr_103.0=0x7c
+port_phy_clause_103.0=0x2d
+port_phy_id0_103.0=0x600d
+port_phy_id1_103.0=0x8500
+portmap_103.0=5:40
+serdes_firmware_mode_103.0=1
+xgxs_rx_lane_map_103.0=0x0123
+xgxs_tx_lane_map_103.0=0x0123
+
+# Port {xe103} [40G only]
+phy_84328_104.0=1
+phy_an_c37_104.0=3
```

Comment:
It may be possible to use phy_an_c37=3 if all ports have the same config. Same for some other configs.